tests/util.py

```diff
@@ -44,5 +49,13 @@ def get_cli_version(self):
         return '0.1.0'
 
     def __init__(self, **kwargs):
-        kwargs['config_dir'] = tempfile.mkdtemp()
+        kwargs['config_dir'] = new_temp_folder()
         super(DummyCLI, self).__init__(**kwargs)
+
+
+def new_temp_folder():
+    temp_dir = os.path.join(tempfile.gettempdir(), TEMP_FOLDER_NAME)
```

Comment:
Does it mean that user cannot run two tests simultaneously?

Reply:
No